static/app/views/performance/landing/widgets/components/widgetContainer.tsx

```diff
@@ -30,64 +30,11 @@ type Props = {
   allowedCharts: PerformanceWidgetSetting[];
   chartHeight: number;
   chartColor?: string;
-  forceDefaultChartSetting?: boolean; // Used for testing.
+  forceDefaultChartSetting?: boolean;
```

Comment:
Does the comment about used for testing no longer apply?

Reply:
No it does, I was too aggressive removing comments, 👍